server/server.js

```
@@ -0,0 +1,45 @@
+const express = require('express');
+const mongoose = require('mongoose');
+const cors = require('cors');
+const path = require('path');
+require('dotenv').config({
+    path: process.env.NODE_ENV === 'production' 
+        ? '.env.production' 
+        : '.env.development'
+});
+const bibleRoutes = require('./routes/bible');
+
+const app = express();
+
+// CORS configuration
+const corsOptions = {
+    origin: process.env.NODE_ENV === 'production' 
+        ? process.env.ALLOWED_ORIGINS?.split(',') || ['https://wordtovictory.github.io']
```

Avoid hard-coding environment and app properties if possible, and fail fast if required properties are missing or invalid.